### ISSUE TITLE
Alternative to mget and lfpt

### ISSUE THREAD

**Issue by sarah872** [flass/pantagruel#22]
I am working on a server and unfortunately, both `mget` and `lfpt` are not installed, and I don't have the rights to install it. Are there alternatives?

**Comment by flass** [comment]
Hi Sarah,

I think there are several answers to your problem:

- regarding installation rights, it's not only `lftp` and `wget` (I suppose that's what you meant, `mget` being a command within `lftp`) that need to be installed, and you may face a lot of trouble setting-up Pantagruel without the admin rights. 
The solution to this would before us to finally release our (extremely belated) Docker image (see #11 ) that would allow you to run the software without having to install anything (else than the Docker client and that particular image).

- regarding the use of `lftp` and `wget` : as these tools rely on connecting to the internet to fetch the data, there might be restrictions set in your sever on outgoing port access. In that respect, there is little I can do as if your server has restricted port access any other software would face the same issue. 
A solution is to download yourself the data and copy them on to the server in the dedicated folders, in which case the download steps during task `00` will be skipped. Options `-a`, `-A` and `-T` allow you to provide your own input data for assemblies and taxonomy, respectively. 
I believe the only other steps that requires internet access are during tasks `03` and `04` to download the UniProt `speclist` file and to check the Interproscan last version, respectively. Task `03` uses the command `wget http://www.uniprot.org/docs/speclist`, but again only if the file is not already present, so you can download it yourself into the folder `03.database/`. And I just made a change (7c67acd) to the task `03` script so that the absence of that file is not an issue. Similarly I made a change to the task `04` script so it does not fail if the version check fails (0c381ad). 
So in summary now you can use Pantagruel without having `lftp` or `wget` installed.

Please let me know if you have further troubles.

Best regards,
Florent
